### PR TITLE
Upgrades ruby version. Bumped gem version

### PR DESCRIPTION
**Pull request by delucas**
In the way of upgrading Nitro to Ruby 2.7, this gem needed an upgrade to be compatible with Ruby 2.7
This change might break retrocompatibility, so we are bumping the minor version.

I'd need comments on the best way to achieve this, so that's why this PR should be considered a RFC with potential to be merged if everything looks OK.

Things I might have done wrong:
1. Update bundler at the same time, but it looked right to have a newer version
2. Update the minor instead of the major semver
3. There is no changelog to upgrade, but that might be useful
4. Add a minor spec for ruby version at 2.7

### CODE REVIEW

**Comment by drborges** [comment]
If these changes are expected to `break retro compatibility` I'd suggest a major version bump instead. Minor version upgrades are usually expected to be retro compatible. @xjunior, @benlangfeld thoughts?

File: ostruct-sanitizer.gemspec [hunk 1]
```diff
@@ -21,7 +21,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.13"
+  spec.required_ruby_version = '>= 2.7'
```
Comment (benlangfeld) [inline ostruct-sanitizer.gemspec:24]: If we're doing this, we can stop testing on old versions.

**Comment by benlangfeld** [comment]
> If these changes are expected to `break retro compatibility` I'd suggest a major version bump instead. Minor version upgrades are usually expected to be retro compatible. @xjunior, @benlangfeld thoughts?

That would be true if we had reached 1.0 already, but we havn't. I'm happy with this version bump and then perhaps cutting a 1.0.0 shortly after.

File: ostruct-sanitizer.gemspec (hunk 1 above)
Comment (xjunior) [reply ostruct-sanitizer.gemspec:24]: Is this necessary?

**Comment by xjunior** [comment]
I thought we had travis watching this repo.

I don't see any reason to require ruby 2.7 if the tests are passing for older versions of ruby.

**Comment by delucas** [comment]
> I thought we had travis watching this repo.
> 
> I don't see any reason to require ruby 2.7 if the tests are passing for older versions of ruby.

@xjunior looks like it's not even running... https://www.travis-ci.com/github/powerhome/ostruct-sanitizer
I'll try to make that work

**Comment by benlangfeld** [comment]
I'd suggest switching to Github Actions.

File: ostruct-sanitizer.gemspec (hunk 1 above)
Comment (delucas) [reply ostruct-sanitizer.gemspec:24]: @xjunior sadly, this change is not retrcompatible

**Comment by delucas** [comment]
This should be ready to be merged and getting a new version released. I can do it all by myself, since this repo doesn't have any restrictions, but I'd feel more comfortable if I could get some code approval. Thanks!
@xjunior @wadewinningham @benlangfeld @drborges

**Review by xjunior — APPROVED** [review]

**Comment by drborges** [comment]
I can cut a new release.

**Pull request merged**